edn/src/types.rs

```diff
@@ -337,14 +345,15 @@ macro_rules! def_common_value_methods {
                 $t::BigInteger(_) => 3,
                 $t::Float(_) => 4,
                 $t::Text(_) => 5,
-                $t::PlainSymbol(_) => 6,
-                $t::NamespacedSymbol(_) => 7,
-                $t::Keyword(_) => 8,
-                $t::NamespacedKeyword(_) => 9,
-                $t::Vector(_) => 10,
-                $t::List(_) => 11,
-                $t::Set(_) => 12,
-                $t::Map(_) => 13,
+                $t::Uuid(_) => 6,
```

Comment:
AIUI it doesn't make a difference, so I was simply putting atoms together, and similar-ish types together. Am I missing some consequence of this?